flow-typed/npm/memoize-one_vx.x.x.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+// flow-typed signature: b6875562f230d378cdab0b00bcdbf926
+// flow-typed version: <<STUB>>/memoize-one_v4.0.0/flow_v0.78.0
+
+/**
+ * This is an autogenerated libdef stub for:
+ *
+ *   'memoize-one'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module 'memoize-one' {
+  declare module.exports: any
```

Comment:
This should probably be sufficient here; haven't tested it but it should be vaguely correct 😅 

```js
declare function memoize<R, A>(fn: (A) => R): (A) => R;
declare function memoize<R, A, B>(fn: (A, B) => R): (A, B) => R;
declare function memoize<R, A, B, C>(fn: (A, B, C) => R): (A, B, C) => R;
declare function memoize<R, A, B, C, D>(fn: (A, B, C, D) => R): (A, B, C, D) => R;
declare function memoize<R, A, B, C, D, E>(fn: (A, B, C, D, E) => R): (A, B, C, D, E) => R;

declare var exports: typeof memoize;
declare export default typeof memoize;
```